src/cuga/backend/cuga_graph/nodes/cuga_lite/cuga_lite_node.py

```
@@ -388,6 +388,23 @@ async def callback_node(
+        # Save trajectory to Kaizen if enabled
+        from cuga.backend.kaizen.kaizen_integration import KaizenIntegration
+        if KaizenIntegration.is_enabled() and state.chat_messages:
+            import asyncio as _asyncio
+            task_id = state.sub_task or tracker.task_id or "unknown"
+            success = not self._has_error(state.final_answer or "")
+            if settings.kaizen.async_save:
+                _asyncio.create_task(
+                    KaizenIntegration.save_trajectory(
+                        state.chat_messages, task_id, success
+                    )
+                )
```

Ruff (0.15.6)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)